wolfhsm/wh_server.h

```diff
@@ -23,6 +24,7 @@ typedef struct CacheSlot {
 } CacheSlot;
 
 typedef struct {
+    int devId;
     curve25519_key curve25519Private[1];
     curve25519_key curve25519Public[1];
     WC_RNG rng[1];
```

Comment:
I don't understand why we have these in the server context.  Leave for now, but we should be sensitive to having extra space here when it may only be needed temporarily.  If these are used mutually exclusively, then consider a union.

Reply:
I agree, though they might have been put there to persist state across cryptoCb invocations? Could use a pointer and a serparate context struct through, or the cryptoCb context for that